OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CQCategory.h
+//  Glycemic Index of Common Foods OptionSelector App
+//
+//  Created by Ayuna Vogel on 8/10/15.
+//  Copyright (c) 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
It's good practice to include at least one comment for each class describing what it does or what data it stores (if it's just a data storage class). Even if it seems obvious to you now, it might not in a month (and it might not to your colleagues).